### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by Pr1yanka**
Updating doc for 3.0.1-preview release

### CODE REVIEW

**Comment by Pr1yanka** [comment]
@JohanFreelancer9 @get-itips  @tiburd  please review

**Review by get-itips — APPROVED** [review]

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
...w.powershellgallery.com/packages/MicrosoftTeams/3.0.1-preview) |<li>Releases modernized versions of Grant-CsTeamsFilesPolicy and Remove-CsTeamsFilesPolicy. These are expected to work similar to their remoting counter parts. The modern versions will be gradually rolled out. So some tenants will see the older remoting version (until the rollout completes).</li><li> Releases [New\|Set\|Remove]-CsUserCallingDelegate cmdlets. These cmdlets will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases [Get\|Remove]-CsTeamsShiftsConnectionErrorReport cmdlets. These cmdlets will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Get-CsTeamsShiftsConnectionOperation cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases New-CsTeamsShiftsConnectionBatchTeamMap cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Remove-CsTeamsShiftsScheduleRecord cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li>
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| December 2021 | [3.0.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/3.0.1-preview) |<li>Releases modernized versions of Grant-CsTeamsFilesPolicy and Remove-CsTeamsFilesPolicy. These are expected to work similar to their remoting counterparts. The modern versions will be gradually rolled out. So some tenants will see the older remoting version (until the rollout completes).</li><li> Releases [New\|Set\|Remove]-CsUserCallingDelegate cmdlets. These cmdlets will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases [Get\|Remove]-CsTeamsShiftsConnectionErrorReport cmdlets. These cmdlets will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Get-CsTeamsShiftsConnectionOperation cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases New-CsTeamsShiftsConnectionBatchTeamMap cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Remove-CsTeamsShiftsScheduleRecord cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li>
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @Pr1yanka, minor suggested change, please ("counter part" to "counterpart". Many thanks.

**Comment by tiburd** [comment]
@rajgodwin 
Hi, Godwin,

The @Pr1yanka GitHub user account isn’t connected to a Microsoft alias, so can you please review this PR and let us know whether you approve of the submitted changes?

For instructions to set up accounts, see:
- [Office authoring guide - getting started - set up accounts](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-github?branch=main)

Thanks!

**Comment by Pr1yanka** [comment]
Hi @tiburd, apologies about that. I have now connected my Microsoft account with this GitHub account. Thank you for the set up link.

**Comment by tiburd** [comment]
@Pr1yanka - thank you! In the future, we recommend submitting PRs in the private repo. The link that you used to set up your account is part of the authoring guide - you can find guidance there for your future contributions.
Thanks again!

**Pull request merged**